api/models.py

```diff
@@ -18,17 +22,17 @@ class Channel(BaseModel):
     '''
     name: str
     id: int | None = None
-    permissions: dict[str, dict[str, bool]] = dict()
+    permissions: dict[str, dict[str, bool]] = {}
```

Comment:
add UTs to make sure all these new models are behaving as expected

Reply:
I'll add UTs in another PR